CMakeLists.txt

```diff
@@ -16,7 +16,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-cmake_minimum_required(VERSION 3.13)
+if(WIN32)
+  cmake_minimum_required(VERSION 3.15)
+else()
+  cmake_minimum_required(VERSION 3.13)
```

Comment:
@ammolitor The centos 6 build uses 3.13.4, and centos 7 installs 3.13.4 for x86 and 3.19.6 for arm.

Can we just upgrade cmake in all the build environments, and delete this #ifdef crud?

Reply:
The windows build require 3.15 minimum for some new VS specific feature, but we are using 3.19 in the docker build.

I *think* the linux build docker image has CMake 3.13 installed so updating it for linux as well would require upgrading the linux CI image.